gopherage/main.go

```diff
@@ -37,7 +38,9 @@ func run() error {
 	rootCommand.AddCommand(diff.MakeCommand())
 	rootCommand.AddCommand(filter.MakeCommand())
 	rootCommand.AddCommand(html.MakeCommand())
+	rootCommand.AddCommand(junit.MakeCommand())
 	rootCommand.AddCommand(merge.MakeCommand())
+	rootCommand.AddCommand(aggregate.MakeCommand())
```

Comment:
This is a merge error - `aggregate` is already at the top of this list. Please remove the duplicate.